.coveragerc

```
@@ -0,0 +1,5 @@
+[run]
+include = xpartamupp/*
+
+[report]
+fail_under = 45
```

```suggestion
branch = yes

[report]
fail_under = 43
```
I'd like to use branch coverage instead of statement coverage. Current branch coverage is just over 43

---

Sounds good. I just added that.